.gitignore

```diff
@@ -7,6 +7,10 @@ yarn-error.log*
 lerna-debug.log*
 .pnpm-debug.log*
 
+# history folder
+
+.history/*
+
```

Comment:
I remember you having all those versioning files in your Homeworks. Just curious - how does that happen, why? Do you have something running on purpose to have them?

Reply:
It is because I often run `git add .` - which is faster for me, instead of adding file by file.
And if you do git add . - usually it will commit history folder as well...